generate_tuxsuite.py

```diff
@@ -36,8 +36,18 @@ def emit_tuxsuite_yml(config, tree, llvm_version):
         print("# This file has been autogenerated by invoking:")
         print(f"# $ ./generate_tuxsuite.py {tree}")
         print("# Invoke tuxsuite via:")
+        patches_flag = patch_series_flag(tree)
         print(
-            f"# $ tuxsuite plan --git-repo {repo} --git-ref {ref} --job-name defconfigs --json-out builds.json {patch_series_flag(tree)}{tuxsuite_yml}"
+            f"# $ tuxsuite plan --git-repo {repo} --git-ref {ref} --job-name defconfigs --json-out builds.json {patches_flag}{tuxsuite_yml}"
+        )
+        print("# Invoke locally via:")
+        print(f"# $ git clone -b {ref} --depth=1 {repo} linux")
+        if patches_flag:
+            patches_folder = patches_flag.strip().rsplit(' ', 1)[-1]
```

Comment:
Mind adding a comment along the lines of:

```python
# Input: foo bar baz
# Output: foobarbaz
```

It's hard for me to imagine what these string operations are doing.

Reply:
Sure thing: 849dfd4916a92d2d2281bb5682b1348f4605fc67